### PR TITLE
Amtool and Alertmanager binaries print to stdout

### PR DESCRIPTION
**Pull request by ArthurSens**
cc @roidelapluie

### CODE REVIEW

File: cmd/alertmanager/main.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func run() int {
 		peerReconnectTimeout = kingpin.Flag("cluster.reconnect-timeout", "Length of time to attempt to reconnect to a lost peer.").Default(cluster.DefaultReconnectTimeout.String()).Duration()
 	)
 
-	promlogflag.AddFlags(kingpin.CommandLine, &promlogConfig)
+	promlogflag.AddFlags(kingpin.CommandLine.UsageWriter(os.Stdout), &promlogConfig)
```
Comment (roidelapluie) [inline cmd/alertmanager/main.go:212]: I'd have that on a dedicated line.

Comment (ArthurSens) [reply cmd/alertmanager/main.go]: done :)

File: cmd/alertmanager/main.go [hunk 2]
```diff
@@ -209,7 +209,8 @@ func run() int {
 		peerReconnectTimeout = kingpin.Flag("cluster.reconnect-timeout", "Length of time to attempt to reconnect to a lost peer.").Default(cluster.DefaultReconnectTimeout.String()).Duration()
 	)
 
-	promlogflag.AddFlags(kingpin.CommandLine, &promlogConfig)
+	a := kingpin.CommandLine.UsageWriter(os.Stdout)
+	promlogflag.AddFlags(a, &promlogConfig)
```
Comment (roidelapluie) [inline cmd/alertmanager/main.go:213]: can't we use kingpin.CommandLine here?

Comment (ArthurSens) [reply cmd/alertmanager/main.go:213]: Sorry I didn't understand what you mean, could you elaborate?

Comment (roidelapluie) [inline cmd/alertmanager/main.go:213]:
```suggestion
	promlogflag.AddFlags(kingpin.CommandLine, &promlogConfig)
	kingpin.CommandLine.UsageWriter(os.Stdout)
```

Comment (ArthurSens) [reply cmd/alertmanager/main.go]: Just a friendly bump to tell that I've commited your suggestion :)

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**